packages/lib/src/form-fields/place-input.wc.svelte

```diff
@@ -0,0 +1,401 @@
+<svelte:options
+  customElement={{
+    tag: 'jp-place-input',
+    shadow: 'none',
+    extend: (customElementConstructor) => {
+      return class extends customElementConstructor {
+        static formAssociated = true;
+
+        constructor() {
+          super();
+          this.attachedInternals = this.attachInternals();
+        }
+      };
+    }
+  }}
+/>
+
+<script lang="ts">
+  import { onMount, createEventDispatcher } from 'svelte';
+  import { clickOutside } from '../clickOutside';
+
+  export let apiKey: String;
+  export let value: string = '';
+  export let attachedInternals: ElementInternals;
+  export let startingLatitude: number = 37.77493;
+  export let startingLongitude: number = -122.41942;
+  export let mapType: 'roadmap' | 'satellite' | 'hybrid' | 'terrain' = 'roadmap';
+  export let zoom: number = 8;
+  export let placeholder: string = '';
+  export let mapId: String;
+  export let returnedValue: 'simple' | 'extended' = 'extended';
+  export let enableMap = true;
+  export let gestureHandling: 'cooperative' | 'greedy' | 'auto' = 'auto';
+  export let types: object = [];
```

Comment:
I think this can be `types: string[]` but it would be even better to define the actual types provided it's a limited list. For instance `types: Array<'hospitals' | 'stations'>`

Reply:
Defining types isn't possible since it is a list with 100+ items, but the link to all the types available is referenced in the documentation.